src/bootstrap.jsx

```diff
@@ -148,8 +148,8 @@ function initializeApi(store) {
     .setUserTokenGetter(() => AuthService.getToken())
     .setUserRequestedGetter(() => AuthService.getUserRequest())
     .setUnauthorizedHandler(() => {
-      if (store.getState().auth.authMode.data.result.mode == authenticationModes.Proxy) {
-        alert('An error has occurred[User unauthorized exception]. Please contact system administrator.');
+      if (!!store.getState().auth.authMode.data && store.getState().auth.authMode.data.result.mode == authenticationModes.Proxy) {
+        window.location = '/_gcp_iap/clear_login_cookie';
```

Comment:
But what if not in IAP authentication mode?

Reply:
**THEN** part of the expression only for PROXY(IAP) mode
**ELSE** part for our standard approach 
so basically for not IAP authentication mode, nothing changed

Reply:
so to be precise the 
IAP redirect you to the Google login page 
not IAP to the arachne/node login page